pkg/build/types/bundle.go

```diff
@@ -46,6 +46,8 @@ type Options struct {
 	// encryption if applicable.
 	// A nil value indicates encryption should not occur.
 	EncryptionKeyInfo *crypt.KeyInfo
+	// ImgCache stores a pointer to the image cache to use.
```

Comment:
Note - this change in order of the struct was needed to satisfy the `malign` linter - so that `bool` are not split by a pointer which creates a memory alignment issue padding required.